.travis.yml

```diff
@@ -21,14 +21,15 @@ services: docker
 env:
   matrix:
   - INSTANCE=ubuntu-1404
-  - INSTANCE=ubuntu-1604
+  # - INSTANCE=ubuntu-1604
```

Comment:
[this isn't the error I remember](https://travis-ci.org/maraaaa/chef-rocketchat/jobs/243893263) although, totally [could be that the mongodb cookbook doesn't converge on 16.04](https://travis-ci.org/maraaaa/chef-rocketchat/jobs/243978396)....

To be fair, I did all of this 21 days ago...  reasons why I did things were stored on `/tmp` have long since been flushed.